src/components/common/TokenImg2.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
```

Comment:
TokenImg2? Where is TokenImg3?

Reply:
I had some commits of the cleanup unpushed, also I have now one thing pending, dependant on the changes of `dex-js` https://github.com/gnosis/dex-js/pull/186

Reply:
I didn't do the refactor since the beginning. I started a component from scratch, based on TokenImage, then when it was working I went for the refactor

The commit that cleaned up the old code was missing. Now is there. Also the reason why travis failed the 1st time